flink-table/flink-table-common/src/main/java/org/apache/flink/table/api/JsonQueryWrapper.java

```diff
@@ -28,7 +28,7 @@
  */
 @PublicEvolving
 public enum JsonQueryWrapper implements TableSymbol {
-    WITHOUT,
+    WITHOUT_ARRAY,
     CONDITIONAL,
```

Comment:
`CONDITIONAL_ARRAY` and `UNCONDITIONAL ARRAY` as well?

Reply:
Hm, yeah. Originally I thought it'd be nice to keep the names more concise, but I guess it's better to stay expressive here.

Reply:
yes, because it is very specific behavior